migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -154,6 +154,13 @@ private void doMigration(MigrationData data) throws IOException {
 
   public void saveProjects(Collection<Project> projects) throws IOException {
     log.info("Migrating projects...");
+    StudyType other = null;
+    for (StudyType st : serviceManager.getStudyDao().listAllStudyTypes()) {
```

Comment:
Should throw an exception if the expected StudyType is not found.

Could also add StudyTypes to ValueTypeLookup and use that to resolve, but since this is only called once per migration, it doesn't make any real difference.